storage/src/mmr/journaled.rs

```diff
@@ -561,12 +571,17 @@ impl<E: RStorage + Clock + Metrics, H: CHasher> Mmr<E, H> {
         self.mem_mmr.root(h)
     }
 
+    /// Process all batched updates without syncing to disk.
```

Comment:
I find this comment confusing. I think it may be better to do something like:

```
Apply all batched updates to the memory-backed MMR.
```

And change the function name to `apply_updates`?

Reply:
Nah, its fine as-is for now. In case it wasn't consistently applied, might've been interesting to debate.